[2026-07-29] Release: Update Apps SDK Starter Kit (#123)
canva-sdks/canva-apps-sdk-starter-kit: CHANGELOG.md

# Changelog

## 2026-07-30

### 🧰 Added

- Added `examples/design_interaction/design_colors` to demonstrate how to retrieve and access design colors with the preview `getDesignColors` API.

> - Added a new example for the requestDataTable available now in beta under the intents package

Reviewer: ```suggestion
- Added a new example for the `requestDataTable` API available now in beta under the intents package.
```

> - Added a new example for the `requestDataTable` API available now in beta under the intents package.

Reviewer: ```suggestion
- Added a new example for the `requestDataTable` API available now in preview under the intents package.
```
We should be using the correct Canva term here.

- Added a new Autofill example demonstrating how to match data fields and autofill the current design using a hardcoded data table.

> - Add example app for brand templates APIs workflow

Reviewer: ```suggestion
- Added a new example app for the Brand Template APIs workflow.
```
